Library/StripeTypes.swift

```diff
@@ -2,11 +2,16 @@ import Foundation
 import KsApi
 
 public protocol StripeSCARequiring {
+  var checkoutId: String? { get }
```

Comment:
What was the motivation for adding `checkoutId` in the `StripeSCARequiring` protocol? From my understanding, `checkoutId` is not actually required to determine SCA eligibility, so it's a bit misleading to add it in here.

Reply:
I'd added there to reuse the existing login on the viewModel, but I guess you're right. I will change that.